packages/@okta/vuepress-site/docs/concepts/session/index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Session management
+meta:
+  - name: description
+    content: An overview of session management in Okta. Learn about sessions in Okta and what kind of session logout to implement.
```

Comment:
replace logout with "sign-in"

Reply:
This wouldn't be hyphenated. Sign in is only hyphenated when it's an adjective - when it's describing a noun...like Sign-In Page or sign-in process. This would just be: ....what kind of session sign in to implement.

Reply:
thanks, i'll replace logout here with "sign out"